src/main/python/deep_qa/data/instances/whodidwhat_instance.py

```diff
@@ -58,12 +58,8 @@ def read_from_line(cls, line: str, default_label: bool=None):
         """
         Reads a WhoDidWhatInstance object from a line.  The format has one of two options:
```

Comment:
Now that we don't have the "left context" and "right context" from the WDW data format, maybe we should rename this to `MCReadingComprehensionInstance`, or something similar?